src/application/application.controller.ts

```diff
@@ -18,10 +18,10 @@ export class ApplicationController {
         return this.applicationService.save(body, req.user);
     }
 
-    // @UseGuards(JwtAuthGuard)
```

Comment:
This would fail sa new applications.. Try to access new application in incognito